lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -12,17 +12,19 @@
 #include "ByteCode/ByteCodeSerializer.h"
 #include "errstr.h"
 #include "ByteCode/ByteCodeDumper.h"
+#include "Codex/Utf8Helper.h"
 #ifdef INTL_WINGLOB
 using namespace Windows::Globalization;
 #endif
```

Comment:
nit: All these endifs are a mess, can you add comments indicating which if they belong to?